src/test-utils.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from "react"
+import { render, RenderOptions } from "@testing-library/react"
+import { ThemeProvider } from "@material-ui/core/styles"
```

Comment:
what is the meaning of this module? You don't import it anywhere, right?

Reply:
This is a custom render method to include theme provider and redux store as wrappers, so that we don't have to wrap the theme provider and redux store provider in each render methods of tests.

https://testing-library.com/docs/react-testing-library/setup/

Yes, I have imported this custom render method in Settings.test.tsx file

Reply:
gotya